src/index.stories.tsx

```diff
@@ -45,13 +46,10 @@ const Instructions = () => (
 // In future, we may want to provide an additional button that can populate the
 // BRF ID on behalf of the user, using our demo link in sandbox.
 //
-export const Base = ({
-  billingRequestFlowID,
-  environment,
-}: {
+export const Base: FC<{
   billingRequestFlowID: string;
   environment: string;
-}) => {
+}> = ({ billingRequestFlowID, environment }) => {
```

Comment:
Same here as mentioned in the previous comment.

Reply:
All done 👍